client/src/partials/debtor_groups/debtor_groups.js

```diff
@@ -0,0 +1,163 @@
+angular.module('bhima.controllers')
+.controller('DebtorGroupsController', DebtorGroupsController);
+
+DebtorGroupsController.$inject = ['$http', '$translate', 'DebtorGroupService', 'SessionService', 'uuid'];
+
+/**
+ * Debtor Groups Controller
+ * This controller is about the debtor groups management module in the admin zone
+ * It's responsible for creating and updating a debtor group
+ * @todo Implementing the delete method for deleting debtor groups
+ */
+function DebtorGroupsController($http, $translate, DebtorGroup, Session, uuid) {
+  'use strict';
+
+  var vm = this;
+
+  /** breadcrumb configurations */
+  vm.bcPaths = [
+    {
+      label: $translate.instant('DEBTOR_GRP.TITLE'),
+      current: true
+    }
+  ];
+  vm.bcButtons = [
+    {
+      dataMethod : 'create',
+      color  : 'btn-default',
+      icon   : 'glyphicon glyphicon-plus-sign',
+      label  : $translate.instant('DEBTOR_GRP.NEW'),
+      action : create
+    }
+  ];
+
+  /** init variables */
+  vm.state = {};
+  vm.debtorGroup = {};
+
+  /** Expose to the view */
+  vm.update = update;
+  vm.cancel = cancel;
+  vm.submit = submit;
+  vm.refreshValidation = refreshValidation;
+
+  /** Load necessary data */
+  debtorGroupsList();
+  accountList();
+  priceList();
+
+  function debtorGroupsList() {
+    return DebtorGroup.read()
+    .then(function (list) {
+      vm.debtorGroupList = list;
+    })
+    .catch(handler);
+  }
+
+  /** @fixme Need service for getting accounts list */
```

Comment:
I'm pretty sure @DedrickEnc created a service to get a list of accounts.  It should be something like `AccountService.read()`.